### PR TITLE
add search banner with tabs

### PR DESCRIPTION
**Pull request by clairesunstudio**
Ticket: https://jira.mass.gov/browse/DP-9086

This PR adds the Tab Molecule and the Search Banner Organism to Mayflower-React. Porting these components that we originally built directly in massgov-search.

1. npm start
2. go to molecules/tabs and confirm that the tab component works/looks as it does in search application. These are dumb components, expecting logic to all be handled in search.
3. go to organisms/searchbanner and confirm the the search banner works/looks as it does in the search application. 

tabs-searchbanner

To use this component will need refactoring in the search repo

### CODE REVIEW

File: src/components/molecules/Tabs/Tabs.stories.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+import { withKnobs, object, select } from '@storybook/addon-knobs/react';
+import { action } from '@storybook/addon-actions';
+
+import Tabs from './index';
+
+storiesOf('molecules', module).addDecorator(withKnobs)
+  .add('Tabs', withInfo('<div></div>')(() => {
+    const tabValues = [
```
Comment (avrilpearl) [inline src/components/molecules/Tabs/Tabs.stories.js]: @clairesunstudio --> can you set these in a options file for the tabs and import it into the story similar to the other components that accept array/objects....

File: src/components/organisms/SearchBanner/index.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          <h2 className="visually-hidden">Search Form</h2>
+          <HeaderSearch {...searchBox} />
+        </div>
+        <Tabs {...tabs} />
+      </div>
+    );
+  }
+}
+
+SearchBanner.propTypes = {
```
Comment (avrilpearl) [inline src/components/organisms/SearchBanner/index.js]: should any of these props be required in this organism

File: src/index.js [hunk 3]
```diff
@@ -70,6 +70,7 @@ export SocialLinks from './components/molecules/SocialLinks';
 export SortResults from './components/molecules/SortResults';
 export ContactGroup from './components/molecules/ContactGroup';
 export ImagePromo from './components/molecules/ImagePromo';
+export Tabs from './components/molecules/Tabs';
 export TypeAheadDropdown from './components/molecules/TypeAheadDropdown';
```
Comment (avrilpearl) [inline src/index.js]: yay! someone remembered to add new components to the export -- woop woop!

File: src/components/organisms/SearchBanner/SearchBanner.stories.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+          },
+          ariaLabel: text('HeaderSearch.buttonSearch.ariaLabel', 'Search'),
+          text: text('HeaderSearch.buttonSearch.text', 'Search')
+        },
+        onSubmit: action('Form submitted'),
+        onChange: action('Text input modified'),
+        defaultText: text('HeaderSearch.defaultText', '')
+      },
+      tabs: {
+        tabs: object('tabs', searchBannerOptions.tabValues),
```
Comment (avrilpearl) [inline src/components/organisms/SearchBanner/SearchBanner.stories.js]: i feel like this should be at the tabs component level...

File: src/components/organisms/SearchBanner/index.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+// eslint-disable-next-line import/no-unresolved
+import './style.css';
+
+class SearchBanner extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchInput: this.props.searchBox.default
+    };
+  }
```
Comment (smurrayatwork) [inline src/components/organisms/SearchBanner/index.js]: The three functions from massgov-search, handleSubmit, handleSearch, and handleTabClick should be preserved. This should be handled by props passed in for each.

Comment (clairesunstudio) [reply src/components/organisms/SearchBanner/index.js]: Should we just pass into handleSubmit --> `searchBox.onSubmit`, handleSearch --> `searchBox.onChange`, handleTabClick --> `tabs.handleClick`?

Comment (smurrayatwork) [reply src/components/organisms/SearchBanner/index.js]: No - HeaderSearch's functions and SearchBanner's functions should be separated.

Comment (smurrayatwork) [reply src/components/organisms/SearchBanner/index.js]: After discussion in slack call, this code is good to go as is.

**Pull request merged**